### PR TITLE
V1.5  JSON: fix `oneOf` documentations

### LINKED ISSUE #257: License documentation broke in JSON schema
JSON documentation for license broke due to #205. Fixing a minor defect in the schema has caused a major defect in the JSON documentation that users consume. May need to revert #205 until such time as we can figure out a way to solve both.

### PR DESCRIPTION
**Pull request by jkowalleck**
fixes #257 without violating JSON schema

non-working "oneOf"s to fix:
- [x]  `components.items.licenses` / `licenseChoice`

### CODE REVIEW

**Comment by jkowalleck** [comment]
CDX_licenses_option1
CDX_licenses_option2

File: schema/bom-1.5.schema.json [hunk 1]
```diff
@@ -915,12 +915,14 @@
...
       "title": "License Choice",
-      "description": "EITHER (a list of SPDX and/or named licenses) OR (a list of one SPDX License Expression)",
+      "description": "EITHER (list of SPDX and/or named licenses) OR (tuple of one SPDX License Expression)",
+      "type": "array",
       "oneOf": [
         {
-          "description": "a list of SPDX and/or named licenses",
+          "title": "Multiple disjunctive licenses",
+          "description": "list of SPDX and/or named licenses",
+          "type": "array",
```
Comment (jkowalleck) [inline schema/bom-1.5.schema.json:925]: adding the `"type": "array",` was necessary to have the docs render properly.

File: schema/bom-1.5.schema.json [hunk 2]
```diff
@@ -931,7 +933,9 @@
           }
         },
         {
-          "description": "a list of one SPDX License Expression",
+          "title": "Single SPDX License Expression",
+          "description": "tuple of one SPDX License Expression",
+          "type": "array",
```
Comment (jkowalleck) [inline schema/bom-1.5.schema.json:938]: adding the `"type": "array",` was necessary to have the docs render properly.

File: schema/bom-1.5.schema.json [hunk 3]
```diff
@@ -915,12 +915,14 @@
       }
     },
     "licenseChoice": {
-      "type": "array",
       "title": "License Choice",
-      "description": "EITHER (a list of SPDX and/or named licenses) OR (a list of one SPDX License Expression)",
+      "description": "EITHER (list of SPDX and/or named licenses) OR (tuple of one SPDX License Expression)",
+      "type": "array",
```
Comment (jkowalleck) [inline schema/bom-1.5.schema.json:920]: This occurrence of `"type": "array",` seamed to have no impact on the rendered docs. 
left it in just in case.

**Review by stevespringett — APPROVED** [review]
Thank you very much for identifying the fixing the issue

**Pull request merged**